recipes/schema-salad/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "schema-salad" %}
+{% set version = "3.0.20181129082112" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 274d7dcc09f4d9fa364ac6f002e614354f4b554a4ca3a7c3dfa3002cd006e7f5
+
+build:
+  number: 0
+  entry_points:
+    - schema-salad-tool=schema_salad.main:main
+    - schema-salad-doc=schema_salad.makedoc:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+
+  run:
+    - cachecontrol >=0.11.7
+    - lockfile >=0.9
+    - mistune >=0.8.1
+    - python
+    - rdflib >=4.2.2
+    - rdflib-jsonld >=0.3.0
+    - requests >=1.0
+    - ruamel.yaml >=0.12.4
+    - setuptools
+    - six >=1.8.0
+    - typing >=3.6.4  # [not py37]
+    - typing_extensions  # [py2k or py36]
+    - future  # [py3k]
+
+test:
+  imports:
+    - schema_salad
+    - schema_salad.tests
+  commands:
+    - schema-salad-tool --help
+    - schema-salad-doc --help
+
+about:
+  home: https://github.com/common-workflow-language/schema_salad
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE.txt
+  summary: Schema Annotations for Linked Avro Data (SALAD)
+  dev_url: https://github.com/common-workflow-language/schema_salad
+
+extra:
+  recipe-maintainers:
+    - croth1
+    - chapmanb
```

Comment:
@chapmanb are you ok with being a maintainer?

Reply:
Great with me, I was the initial maintainer on bioconda and happy to help here as well. Thanks much.